carts/views.py

```diff
@@ -32,3 +32,32 @@ def get(self, request):
 
         except Cart.DoesNotExist:
             return JsonResponse({"message" : "CART_NOT_EXIST"}, status = 404)
+    
+    @login_decorator
+    def patch(self, request):
+        try:
+            cart_id  = request.GET.get('cart_id')
+            data     = json.loads(request.body)
+            quantity = data['quantity']
+
+            if cart_id:
+                Cart.objects.filter(id = int(cart_id)).update(quantity = quantity)
+        
+            return JsonResponse({"message" : "SUCCESS"}, status = 201)
+        
+        except Cart.DoesNotExist:
+            return JsonResponse({"message" : "INVALID_CART"}, status = 404)
+
+    @login_decorator
+    def delete(self, request):
+        try:
+            cart_id  = request.GET.get('cart_id')
+            
+            if cart_id:
+                Cart.objects.filter(id = int(cart_id)).delete()
+            
+            return JsonResponse({"message":"SUCCESS"}, status=204)
```

Comment:
```suggestion
            return JsonResponse({"message":"No content"}, status=204)
```
204의 의미는 성공보다는 다음과 같이 수정하는 것이 좋겠습니다.

Reply:
메세지 내용 수정했습니다.